src/platform/datapath_winuser.c

```diff
@@ -2942,7 +2955,7 @@ RioRecvBufferAllocate(
     if (RecvContext != NULL) {
         RecvContext->RioBufferId =
             Datapath->RioDispatch.RIORegisterBuffer(
-                RecvContext,
+                (char*)RecvContext,
```

Comment:
It requires a **signed** `char*`? We should update this to take a `void*`.

Reply:
Yeah, it's a plain `char*` which is not necessarily signed, at least in terms of idiom.

I'm on the fence on this one. Yes, any pointer works, but forcing a cast to `char*` will provoke at least a bit of thought when calling this function. It certainly made me think.